core/database_util.go

```diff
@@ -36,34 +36,71 @@ var (
 	headBlockKey  = []byte("LastBlock")
 	headFastKey   = []byte("LastFast")
 
-	blockPrefix    = []byte("block-")
-	blockNumPrefix = []byte("block-num-")
+	headerPrefix        = []byte("header-")       // headerPrefix + num (uint64 big endian) + hash -> header
+	tdSuffix            = []byte("-td")           // headerPrefix + num (uint64 big endian) + hash + tdSuffix -> td
+	headerNumSuffix     = []byte("-num")          // headerPrefix + num (uint64 big endian) + headerNumSuffix -> hash
```

Comment:
Naming this only `numSuffix` would probably make it more consistent with `tdSuffix`.